multi-node/emulate-multi-node.md

```diff
@@ -94,9 +94,6 @@ The full setup instructions follow:
  
  ```bash
  $ cat test2.py
- ```
-
```

Comment:
I suppose its a matter of preference, but including the output from `$ cat test2.py` in the same \`\`\`bash [...]\`\`\` block will break the `python` syntax highlighting in the file contents